<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="Renderer" name="GLRenderer" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFIle">
    <property name="title">
     <string>FIle</string>
    </property>
    <addaction name="actionOpen_2"/>
    <addaction name="actionFull_screen_2"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp_2"/>
   </widget>
   <addaction name="menuFIle"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionFull_screen"/>
   <addaction name="actionHelp"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <activeon>:/new/prefix1/images/Actions-help-about-icon.png</activeon>
    </iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="toolTip">
    <string>show help</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/Folder-Open-icon.png</normaloff>:/new/prefix1/images/Folder-Open-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>open a grammer file for visualization</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/Close-2-icon.png</normaloff>:/new/prefix1/images/Close-2-icon.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>exit the application</string>
   </property>
  </action>
  <action name="actionFull_screen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/window-full-screen-icon.png</normaloff>:/new/prefix1/images/window-full-screen-icon.png</iconset>
   </property>
   <property name="text">
    <string>Full screen</string>
   </property>
   <property name="toolTip">
    <string>open the visualization in full screen</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/images/Folder-Open-icon_small.png</normaloff>:/new/prefix1/images/Folder-Open-icon_small.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionFull_screen_2">
   <property name="text">
    <string>Full screen</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionHelp_2">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Renderer</class>
   <extends>QWidget</extends>
   <header>renderer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
